.travis.yml

```diff
@@ -13,16 +13,21 @@ virtualenv:
   system_site_packages: true
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
```

Comment:
one step at a time :P first we want the documentation to be up to date. 
Once done why not translate the doc with Tranifex